src/move/emojicoin_dot_fun/sources/emojicoin_dot_fun.move

```diff
@@ -92,6 +93,8 @@ module emojicoin_dot_fun::emojicoin_dot_fun {
         clamm_virtual_reserves: Reserves,
         cpamm_real_reserves: Reserves,
         lp_coin_supply: u128,
+        cumulative_base_volume: u128,
+        cumulative_quote_volume: u128,
```

Comment:
Just curious for my understanding since I've not used them much, why the usage of Aggregators in `Registry` but not per `Market`? I understand `Registry` will have significantly more resource contention but is there a downside to using aggregators, or is this mostly just an arbitrary decision?